packages/package-json-lint/src/issues/LintIssue.js

```diff
@@ -0,0 +1,28 @@
+const chalk = require('chalk');
+const logSymbols = require('log-symbols');
+
+class LintIssue {
+  constructor({
+    lintId, severity, node, lintMessage,
+  }) {
+    this.lintId = lintId;
+    this.severity = severity;
+    this.node = node;
+    this.lintMessage = lintMessage;
+  }
+
+  /**
+   * Helper to convert the LintIssue to a printable string
+   * @returns {string} Human-friendly message about the lint issue
+   */
+  toString() {
```

Comment:
I think this class could just be a helper function that returns a string, but i don't know if there is a good reason to pick one over the other 🤷‍♂️

Reply:
This will be resolved by my big refactoring.